.devcontainer/gpu/devcontainer.json

```diff
@@ -21,8 +21,9 @@
         "gpu": true
     },
     "runArgs": [
+        "--shm-size=4gb",
         "--gpus=all"
     ],
     "overrideCommand": true,
-    "postStartCommand": "pip install -e ."
+    "postStartCommand": "pip install -e .[dev]"
```

Comment:
Revert these changes. They're outside the scope of your issue and they are handled by us.